frontend/src/profile/Profile.tsx

```diff
@@ -189,6 +186,7 @@ const ProfileComponent: React.FC = () => {
             value={val}
             onChange={(event: React.SyntheticEvent<{}>, value: any) => {
               setMajor(value);
+              setCoopCycle("");
```

Comment:
I think we should be setting it to null here

Reply:
Yeah we should refactor this screen slightly to use nulls instead of empty strings. The main motivation for this is instead of saving a blank string in the db, we save null which is more reflective of "this user has no ___ set". I think we do this in AddPlanPopper and/or EditPlanPopper